core/trino-spi/src/main/java/io/trino/spi/block/AbstractVariableWidthBlock.java

```diff
@@ -23,9 +23,9 @@
 public abstract class AbstractVariableWidthBlock
         implements Block
 {
-    protected abstract Slice getRawSlice(int position);
+    public abstract Slice getRawSlice(int position);
```

Comment:
I'm afraid this totally breaks encapsulation. I don't think maintainers will aproove that